src/dftbp/dftbplus/initprogram.F90

```diff
@@ -1748,6 +1764,45 @@ subroutine initProgramVariables(this, input, env)
         call ThirdOrder_init(this%thirdOrd, thirdInp)
         this%cutOff%mCutOff = max(this%cutOff%mCutOff, this%thirdOrd%getCutOff())
       end if
+
+      ! Initialize multipole module
+      this%tQuadrupole = .false.
+      this%isMdftb = input%ctrl%isMdftb
+      if (this%isMdftb) then
+        @:ASSERT(this%tSccCalc)
+        this%tQuadrupole = .true.
+
+        if (this%tPeriodic) then
+          call error("Multipole expansion currently unsupported for periodic systems")
+        end if
+        if (this%tExtChrg) then
+          call error("Multipole expansion currently unsupported for external charges")
+        end if
+        if (input%ctrl%tSpin) then
+          call error("Multipole expansion currently unsupported for spin-polarised calculations")
```

Comment:
```suggestion
          call error("DFTB multipole expansion currently unsupported for spin-polarised calculations")
```

Reply:
Probably this is actually safe, as the multipole terms only apply to the spin free part.